omod/src/main/java/org/openmrs/module/owa/servlet/RedirectServlet.java

```diff
@@ -51,6 +51,6 @@ private void processRequest(HttpServletRequest request, HttpServletResponse resp
 	        throws IOException {
 		String url = request.getRequestURL().toString().replace("/ms/owa/redirectServlet", "/owa");
 		//@TODO redirecting to original url after login in openmrs.
-		response.sendRedirect(request.getContextPath() + "/login.htm?redirect=" + url);
```

Comment:
Do we need this when we already have the OwaUtils.checkIfAddonManager() check above?

Reply:
@dkayiwa 
This section is in the event that the check didn't yield true for the presence of "owa/addonmanager" in the url. The `redirectServlet` is accessed instead of the `fileServlet`

The `checkIfAddonManager()` method allows the OWAs to be loaded by the fileServlet if the url contains addonmanager